src/shellext/mountmgr.h

```diff
@@ -1,9 +1,17 @@
 #pragma once
 
+// Workaround for compilers without C++17 Standard support.
+// #define NO_CPP17
```

Comment:
There is a standard way to detect the compiler C++ version
https://stackoverflow.com/questions/38456127/what-is-the-value-of-cplusplus-for-c17

Reply:
Yeah, I had read about `__cplusplus`.
Nonetheless, I preferred to keep it "simple" at first, due to
https://docs.microsoft.com/en-us/cpp/build/reference/zc-cplusplus?view=vs-2017
If Mark agrees to check/add that compiler option, then I would fully concur.